examples/README.md

```diff
@@ -0,0 +1,110 @@
+# Examples Description #
+
+## Sql Example ##
+
+Source: [SqlExample.php](./SqlExample.php)
+
+This example shows primary APIs to use with GridGain as with an SQL database:
```

Comment:
Wording looks weird to me, I would say something along the lines of `This example demonstrates GridGain usage as an SQL database`